src/Utils/Utils.flix

```diff
@@ -30,4 +30,11 @@ namespace Utils {
             x
         } else x
     }
+
+    namespace Option {
+        pub def unsafeGet(o: Option[a]): a = match o {
```

Comment:
I think there's another copy of this function somewhere.

Reply:
Also let's tag this `@Unsafe`

Reply:
> I think there's another copy of this function somewhere.

I don't think so? you mean in std lib?

Reply:
I thought I wrote something like this but maybe it was somewhere else.